examples/ammonia_fit_example.py

```diff
@@ -1,7 +1,8 @@
 import pyspeckit
+import astropy.units as u
 
 # Grab a .fits spectrum with a legitimate header
-sp = pyspeckit.Spectrum('../pyspeckit/tests/G031.947+00.076_nh3_11_Tastar.fits')
+sp = pyspeckit.Spectrum('G031.947+00.076_nh3_11_Tastar.fits')
```

Comment:
It looks like you've fixed all of the examples to work in the appropriate path?  That's good, I'm just commenting that I've seen it.  The relative paths that you're removing are things I had to add over the weekend.